boofuzz/monitors/network_monitor.py

```diff
@@ -0,0 +1,29 @@
+from .imonitor import IMonitor
+from . import pedrpc
+
+# Important: IMonitor needs to come *after* pedrpc.Client in the
+# Inheritance list for the method resolution order to produce
+# correct results.
+
+
+class NetworkMonitor(IMonitor, pedrpc.Client):
+    def __init__(self, host, port):
+        super(pedrpc.Client, self).__init__(host, port)
```

Comment:
PyCharm complains about a missing call to __init__ of the super class. I guess because of IMonitor, which doesn't even have a constructor... not sure what to do about this.

Reply:
We can always just add one, even if it does nothing. Or just ignore it. I'm fine with both.

Reply:
Let's just add it so we don't need another ignore.

Also, PyCharm flags `host` and `port` as unexpected arguments. No idea why...

Reply:
Static analysis on dynamic languages are always fun.